### ISSUE TITLE
[a11y] A11y  - Checklist 

### ISSUE THREAD

**Issue by aniamusial** [DivanteLtd/storefront-ui#629]
We need to take care of accessibility, so here's a checklist you can have a look at before creating a pull request with the new feature. Also, these are things that need to be validated at the current state of the project. 

**Keyboard**

- [ ] Focus Visible
Focus styles are needed for navigating with the keyboard to see where we currently are on the page. Remember that elements that are not visible/hidden should not be focusable. All controls, buttons and other interactive elements should have `focus` styles

All interactive Elements should be focusable (and preferably have an intuitive behavior). 
Especially:

SfAccordion
Sliders - SfSlidingSection
SfPagination
Menus - SfMegaMenu, SfHeader

Based on: 
[2.4.7](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html)
[2.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-order.html)

Proposed solutions:
- As we use hover for many components then maybe we can use the same styles for focus when using keyboard?
- We can design a custom outline on focus.
- Using official polyfill  `focus-visible` for keyboard-only navigation: [Click](https://dev.to/wakooka/focus-style-for-keyboard-navigation-only-40ln)

**Images and SVGs**

- [ ] All `img` elements should have `alt `attribute
- [ ] Ensure that `img` elements with `svg` source have` role="img"` attribute
- [ ] All SVGs should have `focusable="false"` when they are the child element of the focusable element.
- [ ] Add `aria-hidden="true"` to decorative `svg`s

Based on: 
[1.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv-all.html)
[1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)
[4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

**Appearance**

- [ ] Contrast ratio -currently in many components we have accessibility errors regarding color contrast ratio. f.e. SfButton, SfBreadcrumbs. This is more of a design issue so we should discuss it further.

Based on: 
[1.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)

**Comment by aniamusial** [comment]
@pixelass please have a look if there is anything else we should add at the current state. Also let's discuss how to style the focus behavior #588

**Comment by pixelass** [comment]
This looks very good. Maybe we could add that 

* All interactive Elements should be focusable (and preferably have an intuitive behavior) 

One of the most common "mistakes" I usually notice are affect following components:

* Range Sliders
* Accordion
* Calendars
* Slideshows
* Menus

**Comment by aniamusial** [comment]
@pixelass Added :)

**Comment by mlama007** [comment]
Make sure selected elements have different style than focused elements
